Jucer2CMake/src/reprojucer.hpp

```diff
@@ -507,6 +507,9 @@ inline void writeReprojucerCMakeLists(const Arguments& args,
       }
     };
 
+  const auto convertSearchPaths = [](const juce::String& searchPaths) -> juce::StringArray
```

Comment:
Since there is no capture, this can be a free function, like `parsePreprocessorDefinitions`. I would call it `parseSearchPaths` since they have a similar role.